app/scripts/modules/core/src/presentation/ToggleButtonGroupInput.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Tooltip } from './Tooltip';
+import { HelpField } from '../help';
+
+export interface IToggleButtonGroupInputProps {
+  propLabel: string;
+  propHelpFieldId?: string;
+  btnGroupSize?: string;
+  tooltipPropOnBtn?: string;
+  displayTextPropOnBtn?: string;
+  tooltipPropOffBtn?: string;
+  displayTextPropOffBtn?: string;
+  onClick: (value: boolean) => void;
+  isPropertyActive?: boolean;
+}
+
+export class ToggleButtonGroupInput extends React.Component<IToggleButtonGroupInputProps> {
+  public static defaultProps: Partial<IToggleButtonGroupInputProps> = {
+    btnGroupSize: 'btn-group-xs',
```

Comment:
I recommend that this is an enumerable `xsmall | small | medium | large` so it is decoupled from the class name.